neqo-transport/src/cc/tests/cubic.rs

```diff
@@ -44,8 +44,15 @@ const fn cwnd_after_loss_slow_start(cwnd: usize, mtu: usize) -> usize {
 /// If false, sets `w_max` lower than cwnd to prevent fast convergence.
 fn setup_congestion_avoidance(
     fast_convergence: bool,
-) -> (ClassicCongestionControl<Cubic>, CongestionControlStats) {
-    let mut cc = ClassicCongestionControl::new(Cubic::default(), Pmtud::new(IP_ADDR, MTU));
+) -> (
+    ClassicCongestionControl<ClassicSlowStart, Cubic>,
```

Comment:
```suggestion
    cc:Classic<ss:Classic, Cubic>,
```
The module is called `cc`, so the `ClassicCongestionControl` -> `cc::Classic` rename should be straightforward. In order to be able to use `ss`, we need a module. (Also wonder if `cc` and `ss` are to terse and we should find better module names.)

Reply:
I don't think `ss` should be it's own module. For me it's the same category as `Cubic` or `NewReno`. I think this just looks very weird because `ClassicSlowStart` is not the best name (because it includes "SlowStart"). 

If it was `ClassicCongestionControl<HyStart, Cubic>` this would look much more sensible imo. I'm not sure what a better name for the standard slow start is though. I'm reluctant towards just naming it `SlowStart` or `Classic` because that might get confusing.

As for `ClassicCongestionControl` I think we should refactor the structure there a bit anyways because right now we don't have anything else then `Classic` and are not planning to implement BBR afaik. So it seems a bit unnecessary to have the `CongestionControl` trait that is only implemented by `ClassicCongestionControl`. I think it could just be `CongestionControl` without an extra trait. But that is a seperate PR and I'd like to focus on getting the functionality in before doing that.

But yes, ideally this would become something like `CongestionController<HyStart, Cubic>` (or any other combination).